all_bin_scenario.py

```diff
@@ -297,33 +290,10 @@ def max_stepsize(t: float) -> float:
             global_data.update(quantities)
 
     ############# RUN DIV BIN SIMUS #############
-    for nb_bin in range(total_fw_bins, 22):
-        sub_bin = Div_bins.get_bin(nb_bin)
-
-        if sub_bin.material == "W":
-            my_model, quantities = make_W_mb_model(
-                temperature=T_function_W,
-                deuterium_ion_flux=deuterium_ion_flux,
-                tritium_ion_flux=tritium_ion_flux,
-                deuterium_atom_flux=deuterium_atom_flux,
-                tritium_atom_flux=tritium_atom_flux,
-                # FIXME: -1s here to avoid last time step spike
-                final_time=my_scenario.get_maximum_time() - 1,
-                L=sub_bin.thickness,
-                folder=f"mb{nb_bin+1}_results",
-            )
-        elif sub_bin.material == "B":
-            my_model, quantities = make_B_mb_model(
-                temperature=T_function_B,
-                deuterium_ion_flux=deuterium_ion_flux,
-                tritium_ion_flux=tritium_ion_flux,
-                deuterium_atom_flux=deuterium_atom_flux,
-                tritium_atom_flux=tritium_atom_flux,
-                # FIXME: -1s here to avoid last time step spike
-                final_time=my_scenario.get_maximum_time() - 1,
-                L=sub_bin.thickness,
-                folder=f"mb{nb_bin+1}_results",
-            )
+    # for div_bin in Div_bins.bins: 
+    for div_bin in range(total_fw_bins, 22): # only running 4 div bins to demonstrate capability 
+        sub_bin = Div_bins.get_bin(div_bin)
+        my_model, quantities = which_model(div_bin, sub_bin.material)
```

Comment:
Similar to comment above:
```suggestion
    for div_bin in Div_bins.bins[:4]: # only running 4 div bins to demonstrate capability 
        my_model, quantities = which_model(div_bin, div_bin.material)
```